config/initializers/say_when.rb

```diff
@@ -0,0 +1,34 @@
+require 'say_when'
+# require 'say_when/poller/concurrent_poller'
+
+# you can specify a the logger
+SayWhen.logger = Rails.logger
+
+# configure the scheduler for how to store and process scheduled jobs
+# it will default to a :memory strategy and :simple processor
+SayWhen.configure do |options|
+  # options[:storage_strategy]   = :memory
+  options[:storage_strategy]   = :active_record
+
+  # options[:processor_strategy] = :simple
+  options[:processor_strategy] = :active_job
+
+  options[:queue] = :feeder_default
+end
+
+begin
+  job = SayWhen.schedule(
```

Comment:
you don't - I was probably assigning it by habit, and b/c at one point I was logging it there